app/helpers/repo_helper.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module RepoHelper
+  require 'rest-client'
+  require 'json'
+
+  def get_projects_of_user(user, page)
+    all_user_projects_url = "https://gitlab.com/api/v4/users/#{user.gid}"\
```

Comment:
When you move this to a service, you could break this down into parts. Like, `base_url`, `version`, `endpoint`.
This will help us adapt to self-hosted Gitlab, different versions, different endpoints etc.